api/api_sources/schema-migration-sql/CountryProvinceSchema/CountryProvinceSchema.sql

```diff
@@ -0,0 +1,37 @@
+-- ### Creating Table: country_province ### --
+
+        
+CREATE TABLE country_province ();
+ALTER TABLE country_province ADD COLUMN province_code VARCHAR(2) NOT NULL;
+ALTER TABLE country_province ADD COLUMN country_code VARCHAR(3) NOT NULL;
```

Comment:
Minor suggestion for column ordering in the table... should put the country_code before province_code. It puts the columns in a more logical order (i.e. Country is the "parent" to the Province/State).

Reply:
Resolved